src/overlays/actors/ovl_Bg_Inibs_Movebg/z_bg_inibs_movebg.c

```diff
@@ -27,17 +26,56 @@ const ActorInit Bg_Inibs_Movebg_InitVars = {
     (ActorFunc)BgInibsMovebg_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B96578[] = {
+extern Gfx D_060062D8[];
+extern Gfx D_06001DC0[];
+extern Gfx D_06006140[];
+extern Gfx D_06001C10[];
+extern AnimatedMaterial D_06006858;
+extern AnimatedMaterial D_06002598;
+Gfx* D_80B96560[] = { D_060062D8, D_06001DC0 };
+Gfx* D_80B96568[] = { D_06006140, D_06001C10 };
+AnimatedMaterial* D_80B96570[] = { &D_06006858, &D_06002598 };
+
+static InitChainEntry sInitChain[] = {
     ICHAIN_VEC3F_DIV1000(scale, 1000, ICHAIN_STOP),
 };
 
-#endif
+void BgInibsMovebg_Init(Actor* thisx, GlobalContext* globalCtx) {
+    BgInibsMovebg* this = THIS;
+
+    Actor_ProcessInitChain(&this->dyna.actor, sInitChain);
+    DynaPolyActor_Init(&this->dyna, 1);
+
+    this->unk_15C = D_80B96560[BGINIBSMOVEBG_GET_F(thisx)];
+    this->unk_160 = D_80B96568[BGINIBSMOVEBG_GET_F(thisx)];
+    this->unk_164 = D_80B96570[BGINIBSMOVEBG_GET_F(thisx)];
+}
+
+void BgInibsMovebg_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    BgInibsMovebg* this = THIS;
+
+    DynaPoly_DeleteBgActor(globalCtx, &globalCtx->colCtx.dyna, this->dyna.bgId);
+}
+
+void BgInibsMovebg_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    BgInibsMovebg* this = THIS;
 
-extern InitChainEntry D_80B96578[];
+    AnimatedMaterial* animMat;
+    Gfx* dl1;
+    Gfx* dl2;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Inibs_Movebg/BgInibsMovebg_Init.s")
+    animMat = this->unk_164;
+    if (animMat != 0) {
```

Comment:
```suggestion
    if (animMat != NULL) {
```
Since `aninMat` is a pointer.